node_modules/gh-core/lib/db.js

```diff
@@ -468,8 +468,32 @@ var _setUpModel = function(sequelize) {
     Serie.belongsTo(Group);
     Group.hasMany(Serie);
 
-    OrgUnit.hasMany(Serie);
-    Serie.hasMany(OrgUnit);
+    /**
+     * The join table between the OrgUnit and Serie tables. This table is defined explicitly,
+     * as Sequelize doesn't intuitively support three way joins.
+     *
+     * This table will hold 3 values:
+     *  - The organisational unit that contains a serie
+     *  - The serie that is in the organisational unit
+     *  - An optional organisational unit where the serie was "borrowed" from
+     *
+     * This is to allow for the timetable use-case where series can be borrowed from each other
```

Comment:
I talked to Sam about this and he said it was fine to show org unit B as that is where the administrator borrowed it from.